[2024-11-05T07:18:46.965+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-05T07:18:47.011+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2024-11-04T00:00:00+00:00 [queued]>
[2024-11-05T07:18:47.043+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2024-11-04T00:00:00+00:00 [queued]>
[2024-11-05T07:18:47.045+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-11-05T07:18:47.115+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2024-11-04 00:00:00+00:00
[2024-11-05T07:18:47.128+0000] {standard_task_runner.py:72} INFO - Started process 214 to run task
[2024-11-05T07:18:47.132+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'scheduled__2024-11-04T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/our-first-dag.py', '--cfg-path', '/tmp/tmp17ihfbk2']
[2024-11-05T07:18:47.135+0000] {standard_task_runner.py:105} INFO - Job 21: Subtask second_task
[2024-11-05T07:18:47.303+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.second_task scheduled__2024-11-04T00:00:00+00:00 [running]> on host f4d9f79457ef
[2024-11-05T07:18:47.551+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Adithya' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-04T00:00:00+00:00'
[2024-11-05T07:18:47.553+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-05T07:18:47.593+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-05T07:18:47.596+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello I am after task1']
[2024-11-05T07:18:47.618+0000] {subprocess.py:86} INFO - Output:
[2024-11-05T07:18:47.621+0000] {subprocess.py:93} INFO - hello I am after task1
[2024-11-05T07:18:47.622+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-05T07:18:47.717+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-05T07:18:47.718+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, run_id=scheduled__2024-11-04T00:00:00+00:00, execution_date=20241104T000000, start_date=20241105T071847, end_date=20241105T071847
[2024-11-05T07:18:47.831+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-05T07:18:47.871+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-05T07:18:47.875+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
